docs/src/transformers.md

```diff
@@ -98,16 +98,29 @@ y1 = predict(ridgeM, W)
 knnM = machine(knn, W, y)
 y2 = predict(knnM, W)
 
-mach = machine(averager)
-yhat = transform(mach, y1, y2)
+mach= machine(averager)
+yhat = transform(averagerM, y1, y2)
 ```
```

Comment:
@OkonSamuel Many thanks for this!

I guess you mean to replace `mach` on line 101 with `averagerM` as well.

I know it's a pain, but I always test all code that isn't automatically run  by `Documenter.jl` to catch such things.  The alternative is to change the "```julia" block to a "```example???" block, but this is also sometimes more hassle, and I had some issues with Documenter.jl early on  (so it's a bit of a mish-mash or evaluated and non-evaluated code blocks at the moment).

Reply:
> I guess you mean to replace `mach` on line 101 with `averagerM` as well.

Yeah. fixed that now.